helm_deploy/hmpps-manage-users/values.yaml

```diff
@@ -22,6 +22,23 @@ generic-service:
         limit_req_status 429;
     tlsSecretName: hmpps-manage-users-cert
     modsecurity_enabled: true
+    modsecurity_snippet: |
+      SecRuleEngine On
+      {{ if .Values.ingress.modsecurity_audit_enabled -}}
+      SecAuditEngine RelevantOnly
+      SecAuditLog /var/log/nginx/error.log
+      SecAuditLogType Serial
+      {{ end -}}
+      SecRuleUpdateActionById 949110 "t:none,deny,status:406,logdata:%{SERVER_NAME}"
+      SecRuleUpdateActionById 959100 "t:none,deny,status:406,logdata:%{SERVER_NAME}"
+      SecRuleUpdateTargetById 942100 "!ARGS"  # disable libinjection testing on request args. to disable all SQLi testing, modify rules in the range 942100-942999
```

Comment:
This is only new line that I needed - the rest is from the generic service chart

Reply:
I don't understand? What does it do? 😆

Reply:
We have had an issue where the POST isn't being allowed through the firewall.  In this thead...[CloudPlatform ](https://mojdt.slack.com/archives/C57UPMZLY/p1659528717441779) it has been highlighted that there is an issue https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/794
It was recommended to add the disable for the libinjection - which I did manually into preprod, but this is the actual change as obviously re-releasing manage-users will overwrite it.